### PR TITLE
#5625 issue fix

### PR DESCRIPTION
**Pull request by SSPkrolik**

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://zeoalliance-issue-5625-fix.c68bfdc4cac4c32c41e1a922e73062ea.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://zeoalliance-issue-5625-fix.c68bfdc4cac4c32c41e1a922e73062ea.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by jvanbaarsen** [comment]
@SSPkrolik Can you also make a regression test for this?

File: app/controllers/profiles_controller.rb [hunk 1]
```diff
@@ -70,4 +73,8 @@ def user_params
       :avatar, :hide_no_ssh_key,
     )
   end
+
+  def quote_bio
```
Comment (jvanbaarsen) [inline app/controllers/profiles_controller.rb]: What about dealing with this in the model?

File: app/controllers/profiles_controller.rb [hunk 2]
```diff
@@ -57,6 +59,7 @@ def update_username
 
   def user
     @user = current_user
+    @user[:bio] = raw @user[:bio]
```
Comment (jvanbaarsen) [inline app/controllers/profiles_controller.rb]: Would it be possible to return a raw bio in the user object?

**Comment by jvanbaarsen** [comment]
This merge request has been closed because a request for more information has not been reacted to for more than 2 weeks. If you respond and conform to the merge request guidelines in our [contributing guidelines](https://gitlab.com/gitlab-org/gitlab-ce/blob/master/CONTRIBUTING.md#pull-requests) we will reopen this merge request.

**Pull request closed without merging**